code/modules/research/techweb/all_nodes.dm

```diff
@@ -1047,6 +1047,7 @@
 	/obj/item/cautery/alien, /obj/item/surgicaldrill/alien, /obj/item/screwdriver/abductor, /obj/item/wrench/abductor, /obj/item/crowbar/abductor, /obj/item/multitool/abductor, /obj/item/stock_parts/cell/infinite/abductor,
 	/obj/item/weldingtool/abductor, /obj/item/wirecutters/abductor, /obj/item/circuitboard/machine/abductor, /obj/item/abductor_baton, /obj/item/abductor, /obj/item/stack/sheet/mineral/abductor)
```

Comment:
thanks to https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9709 this just makes the upgrade an expensive, but otherwise easily obtained timegate. you don't need antag gear for it.